features/step_definitions/common_steps.rb

```
@@ -258,7 +258,7 @@
 end
 
 Then /^I see a validation message containing '(.*)'$/ do |message|
-  validation_message = page.find(:css, ".validation-message")
+  validation_message = page.find(:css, ".validation-message, .govuk-error-message")
```

What happens if we end up finding both? (I doubt it will happen in practice, just curious)

---

I'm not really sure... my gut feeling says it should work, because there could also be instances with multiple validation messages; but then I don't know if we test any of those.

I'll do some digging.

---

`page.find` returns the first found, so if there are two and the first one found has the correct text, the test will pass.

---

It's not documented, but `page.find` will raise an error if it finds more than one match :D

---

OK! I'd rather it fail noisily as above than giving a false positive with the wrong message, if you see what I mean.